### DISCUSSION TITLE
Updating NSpanel to 12.9 of ESPHOME

### DISCUSSION THREAD

**Discussion by rploeg** [Blackymas/NSPanel_HA_Blueprint#1795] [Q&A]
On three of my panels I get this error when I try to update the device with latest ESPhome

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/49752333/11a8f70d-2e0c-4b87-ac91-4d1f8d025aad)

Any idea what is wrong? I am running latest HA and ESPhome and 4.1.4 of this solution

**Comment by edwardtfn** [comment]
Try cleaning build files: https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/error_compiling.md
